service-commander-def.yaml

```diff
@@ -3,6 +3,6 @@ dir: .
 start_cmd: ../../bin/mapepire
 check_alive: mapepire,8076
 batch_mode: 'true'
-sbmjob_opts: 'JOBQ(QUSRNOMAX) ALWMLTTHD(*YES)'
+sbmjob_opts: 'JOBQ(QUSRNOMAX)'
```

Comment:
@ThePrez Removed this parameter since it already gets put in the command.

https://github.com/Mapepire-IBMi/mapepire-server/actions/runs/10531417456/job/29183416357#step:15:412

```
CPYENVVAR(*YES) PRTDEV(*USRPRF) ALWMLTTHD(*YES) JOB(MAPEPIRED) JOBQ(QUSRNOMAX) ALWMLTTHD(*YES)  <
child process mapepired:CPD0042: Parameter ALWMLTTHD specified more than once.
```